packages/bpk-component-text/stories.js

```diff
@@ -90,4 +90,21 @@ storiesOf('bpk-component-text', module)
       The man jumped over the shark tank. That was very{' '}
       <BpkText bold>bold</BpkText> indeed.
     </BpkText>
+  ))
+  .add('using in a SVG', () => (
```

Comment:
We don't really need this story as we are only adding a new tag name. Can this be removed please 😄